Modules/System/BLOB Storage/src/TempBlobList.Codeunit.al

```diff
@@ -67,6 +67,18 @@ codeunit 4102 "Temp Blob List"
         exit(TempBlobListImpl.RemoveAt(Index));
     end;
 
+    /// <summary>
+    /// Remove a number of elements starting from a specified location from a non-empty list.
+    /// </summary>
+    /// <error>The index is larger than the number of elements in the list or less than one.</error>
+    /// <param name="Index">The index of the first element to be deleted.</param>
+    /// <param name="ElementCound">The number of elements to be deleted.</param>
```

Comment:
Typo

```suggestion
    /// <param name="ElementCount">The number of elements to be deleted.</param>
```

Reply:
Also, what if negative value is passed? Is this supported?

Reply:
Thanks for the comment. Added an explicit error and test cases for a negative count.